### PR TITLE
Explicitly Stop Referencing Custom Database Instance ParameterGroup

### PR DESCRIPTION
**Pull request by sureshc**
In #57356 we stopped explicitly setting a DB Instance ParameterGroup, but a defect in CloudFormation appears to leave existing database instances still using the custom Instance ParameterGroup after that Pull Request is deployed.

Once #57356 has been applied to staging, test, levelbuilder, and production, merge this change to explicitly use the default Instance ParameterGroup. This will clear all usage of the Aurora 2 InstanceParameter Group and will allow us to delete that Resource in the future after the upgrade to Aurora 3.

## Testing story

Already tested in the tests for #57356

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: aws/cloudformation/components/database.yml.erb [hunk 1]
```diff
@@ -134,6 +138,11 @@
         # BEGIN: Dynamic configuration settings. Changing these settings does NOT require nor trigger restart of instances.
         slow_query_log: 1
         innodb_monitor_enable: all
+        tx_isolation: READ-COMMITTED
+
+        # Prevent a slow query from degrading instance or cluster performance.
+        # https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_execution_time
+        max_execution_time: 30000
```
Comment (cat5inthecradle) [inline aws/cloudformation/components/database.yml.erb:145]: `tx_isolation` vs `transaction_isolation` below. Is that an intentional difference?

**Review by cat5inthecradle — COMMENTED** [review]
All of the changes to the actual parameter groups aren't mentioned in the description of the PR. What are the consequences of those changes?

**Comment by sureshc** [comment]
This started out as a feature branch off of a feature branch and I think the commit history and git diffs are now corrupted. The only change in this feature branch should be:

`DBParameterGroupName: default.aurora-mysql5.7 #Explicitly stop setting a custom Instance ParameterGroup.`

**Comment by sureshc** [comment]
This started out as a feature branch off of a feature branch and I think the commit history and git diffs are now corrupted. The only change in this feature branch should be:

`DBParameterGroupName: default.aurora-mysql5.7 #Explicitly stop setting a custom Instance ParameterGroup.`

I'll close this Pull Request and re-create it with a fresh branch.

**Pull request closed without merging**